va/va.h

```diff
@@ -2044,6 +2044,15 @@ typedef enum {
     */
     VAEncryptionParameterBufferType = 60,
 
+    /**
+     * \brief Encoding Delta QP buffer
+     *
+     * This buffer contains delta QP per block for encoding. Currently
+     * VAEncDeltaQPBufferType is defined for HEVC encoding, see
+     * #VAEncDeltaQPBuffer for details
+     */
+    VAEncDeltaQPBufferType                   = 61,
```

Comment:
For the encoder is this meant to be an input or an output or both?
As in input it sounds like QP deltas to use when encoding, like a more complex but finer-grained way of doing ROI.  As an output it sounds like the QP deltas which got applied by BRC.

Reply:
This is an input buffer